### PR TITLE
Add `macosArm64` target to `log-engine-tuulbox`

### PR DESCRIPTION
**Pull request by Phoenix7351**
Adding macosArm64 target for mac M1 usage.

### CODE REVIEW

**Review by cedrickcooke — APPROVED** [review]

File: log-engine-tuulbox/build.gradle.kts [hunk 1]
```diff
@@ -14,10 +14,13 @@ kotlin {
         publishAllLibraryVariants()
     }
     js().browser()
+
     iosX64()
     iosArm32()
     iosArm64()
     macosX64()
+    macosArm64()
+    iosSimulatorArm64()
```
Comment (twyatt) [inline log-engine-tuulbox/build.gradle.kts:23]: Have you tested that BLE is available on the simulator?

Comment (twyatt) [inline log-engine-tuulbox/build.gradle.kts:23]: Have you tested that BLE is available on the simulator?

File: log-engine-tuulbox/build.gradle.kts [hunk 2]
```diff
@@ -14,10 +14,13 @@ kotlin {
         publishAllLibraryVariants()
     }
     js().browser()
+
     iosX64()
     iosArm32()
     iosArm64()
     macosX64()
+    macosArm64()
```
Comment (twyatt) [inline log-engine-tuulbox/build.gradle.kts:22]: I'd like this to be tested before merging it.
Do you need me to test it, or can you try this out on the sensortag sample app?

File: log-engine-tuulbox/build.gradle.kts (hunk 1 above)
Comment (Phoenix7351) [reply log-engine-tuulbox/build.gradle.kts:23]: I have not come remotely close to that. Our other repo still needs to be updated to produce a swift package that iOS can consume. I just wanted to get everything building first.

Comment (Phoenix7351) [reply log-engine-tuulbox/build.gradle.kts:23]: As above, not yet.

File: log-engine-tuulbox/build.gradle.kts (hunk 2 above)
Comment (Phoenix7351) [reply log-engine-tuulbox/build.gradle.kts:22]: My sensortag died about a year ago. If you can test it faster than me getting a new one, I'd say go for it.

File: log-engine-tuulbox/build.gradle.kts (hunk 1 above)
Comment (Phoenix7351) [reply log-engine-tuulbox/build.gradle.kts]: Removed!

Comment (Phoenix7351) [reply log-engine-tuulbox/build.gradle.kts]: Removed!

**Review by twyatt — COMMENTED** [review]
Please re-request review after testing with [SensorTag](https://github.com/JuulLabs/sensortag#macos) on M1.

**Comment by Phoenix7351** [comment]
@twyatt Tested on David's machine with your Sensortag and it works just fine. We have no idea why it's not working on my machine. I can't see any BLE advertisements. 😭

**Review by twyatt — APPROVED** [review]

File: log-engine-tuulbox/build.gradle.kts (hunk 1 above)
Comment (twyatt) [reply log-engine-tuulbox/build.gradle.kts]: I didn't realize that we already have `iosSimulatorArm64()` in `core`. Added via a PR that an external contributed provided.

For consistency, we should probably have it here.

Sorry, for the back-and-forth.

**Pull request merged**